### DISCUSSION TITLE
NameError: name 'example_list' is not defined

### DISCUSSION THREAD

**Discussion by alaminsayed** [mrdbourke/pytorch-deep-learning#1160] [Q&A]
![Screenshot 2024-12-30 194940](https://github.com/user-attachments/assets/bca1788f-0a2a-47ac-a3a1-fd78ed04fbd5)
its from foodvision_mini_video.i tried to do with my own dataset..and trying to upload it on huggingface..but this error is showing,,

**Comment by fawern** [comment]
Hi @alaminsayed 

The error is because example_list hasn't been defined. You can fix it by defining it before your gr.Interface like this:

```python
example_list = [
    ["paht/image1.jpg"],
    ["path/image2.jpg"]
]
```

Make sure you run this code before calling gr.Interface, and then pass example_list as the examples argument.

That should solve the issue.

Good luck
